opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -220,9 +219,10 @@ message Span {
   // Status.Ok (code = 0).
   Status status = 16;
 
-  // An optional number of child spans that were generated while this span
-  // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 17;
+  // An optional number of local child spans that were generated while this span
```

Comment:
Can you add a small note on how consumer can use this information. I.e. semantics of this field beyond just description. Or link to spec describing it?